book/invalidation.md

```diff
@@ -1706,7 +1708,8 @@ First, there's a lot of `style` recomputation:
     Change ProtectedField(<header>, style)
     Change ProtectedField(<h1 class="title">, style)
     Change ProtectedField('Reusing Previous Computations', style)
-    Change ProtectedField(<a href="https://twitter.com/browserbook">, style)
+    Change ProtectedField(
+        <a href="https://twitter.com/browserbook">, style)
```

Comment:
```suggestion
    Change ProtectedField(<a href="...">, style)
```

Since this is supposed to be terminal output, not code, I'm wary of adding whitespace. Let's just elide the boring part.